dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -185,6 +185,16 @@ void DAGQueryBlockInterpreter::handleMockTableScan(const TiDBTableScan & table_s
         analyzer = std::make_unique<DAGExpressionAnalyzer>(std::move(names_and_types), context);
         pipeline.streams.insert(pipeline.streams.end(), mock_table_scan_streams.begin(), mock_table_scan_streams.end());
     }
+    std::vector<std::pair<UInt64, DataTypePtr>> generated_column_infos;
+    for (Int64 i = 0; i < table_scan.getColumnSize(); i++)
+    {
+        if (table_scan.getColumns()[i].hasGeneratedColumnFlag())
+        {
+            auto data_type = getDataTypeByColumnInfoForComputingLayer(table_scan.getColumns()[i]);
+            generated_column_infos.push_back(std::make_pair(i, data_type));
+        }
+    }
+    executeGeneratedColumnPlaceholder(pipeline.streams.size(), generated_column_infos, log, pipeline);
```

Comment:
Looks like it also needs to be added in `PhysicalMockTableScan`

Reply:
Maybe just ignore this for `PhysicalMockTableScan` and `handleMockTableScan`.
Because we have not unit-test for generated column.

Reply:
lgtm